noxfile-template.py

```diff
@@ -76,11 +140,25 @@ def lint(session):
     session.install("flake8", "flake8-import-order")
 
     local_names = _determine_local_import_names(".")
-    args = FLAKE8_COMMON_ARGS + [
+    options = [
         "--application-import-names",
-        ",".join(local_names),
-        ".",
+        ",".join(local_names)
     ]
+
+    # We currently look at pytest.ini for flake8 config.
+    # You can add your own exclude and ignore by using `extend-`
+    #
+    # Example config:
+    # [flake8]
+    # extend-ignore = I100
+    # extend-exclude = myapp1,myapp2
+    if os.path.isfile("pytest.ini"):
+        options += [
+            "--append-config",
+            "pytest.ini",
+        ]
+    options.append(".")
+    args = FLAKE8_COMMON_ARGS + options
```

Comment:
nit: IMO, I don't think we should let style checks be configurable, but we can discuss on the next sync

Reply:
I agree in general, but it's required for the cloud-run case.

Example:
```
my_proj
\
 e2e_test.py
 requirements.txt
 my_app1
 \
  main.py
  main_test.py
  requirements.txt
```

The lint session for the parent my_proj will report that `import main` is in the wrong place because it doesn't recognize the local import files correctly.

So in this case having following `pytest.ini` is the cleanest solution IMO:

```
[pytest]

norecursedirs =  *

[flake8]
extend-exclude = myapp1
```
This files allows to skip subdirectories both for pytest and flake8.